### PR TITLE
Eda

### PR DESCRIPTION
**Pull request by lucjaesterastaniek**
Please feel free to comment!

### CODE REVIEW

File: final_eda.py [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+#SCALING DATA
+
+#StandardScaler
+scaler = StandardScaler()
+
+X_train_scale = StandardScaler().fit_transform(X_train)
+
+#MinMaxScaler
+minmaxscaler = MinMaxScaler()
+X_train_minmaxscaler = MinMaxScaler().fit_transform(X_train)
```
Comment (opopiol) [inline final_eda.py:158]: You can add a function to this two scalers

File: final_eda.py [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+#----------------------------------------------------------------------------------
+#PCA
+
+sns.set_palette('Set1')
+
+def pca_scatterplot(n):
+  """This function returns scatterplot
```
Comment (opopiol) [inline final_eda.py:166]: You can make this more specific by adding what this scatterplot display

File: final_eda.py [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+#----------------------------------------------------------------------------------
+#DISTRIBUTION
+def distribution(x_1: np.array):
+    """This function returns plot with distribution
+    :param x_1: np.array: train data after splitting
+    :return: sns.kdeplot(pd.DataFrame(x_1)[i]): plot with distribution
+    """
+  for i in [x_1.skew().idxmin(), x_1.skew().idxmax()]:
+    sns.kdeplot(pd.DataFrame(x_1)[i])
+    plt.legend(labels=['x_1.skew().idxmin()', 'x_1.skew().idxmax()'])
```
Comment (opopiol) [inline final_eda.py:141]: You can add tittle to this plot

**Review by opopiol — COMMENTED** [review]
Apart from a few comments, the code is fully functional

File: final_eda.py (hunk 1 above)
Comment (lucjaesterastaniek) [reply final_eda.py:158]: Good point! I'll try to get them in order.

**Comment by lucjaesterastaniek** [comment]
Adding a title, closing a few lines of code in a function and adding an extended description. Thanks for the tips!

**Pull request closed without merging**